pandas/tests/frame/test_indexing.py

```diff
@@ -2626,6 +2626,20 @@ def test_index_namedtuple(self):
         result = df.loc[IndexType("foo", "bar")]["A"]
         assert result == 1
 
+    def test_index_single_double_tuples(self):
+        # GH 20991
+        tuple_1 = tuple([1, 2])
+        tuple_2 = tuple([1])
+        df = DataFrame([[tuple_1], [tuple_2]], columns=["A"]).set_index("A")
+
+        result = df.loc[[df.index[0]]]
```

Comment:
`df.loc[tuple_1]`; if you can avoid `df.index[...]` calls here will make the test more concise, as accessing index values like that isn't related to the test

Reply:
Thanks @WillAyd  - removed `.index[...]`